tests/Core/Tokenizer/ExplicitOctalNotationTest.inc

```diff
@@ -0,0 +1,4 @@
+<?php
+
+/* testExplicitOctal declaration */
```

Comment:
I'd remove `declaration` from this comment for consistency with other test files